### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by PsiACE**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Just a normal maintenance. 

The octocrab has been removed here because we no longer have a GitHub engine.

The workspace-hack will be retained and will allow us to introduce other tool hacks later.

Fixes #issue

### CODE REVIEW

**Comment by PsiACE** [comment]
cc @Xuanwo Can we remove the http-types patch now?

**Comment by Xuanwo** [comment]
> cc @Xuanwo Can we remove the http-types patch now?

Let's remove it.

File: src/workspace-hack/Cargo.toml [hunk 1]
```diff
@@ -4,11 +4,5 @@ version = "0.1.0"
 edition = "2021"
 
 [package.metadata.cargo-udeps.ignore]
-normal = ["openssl-src", "axum"]
 
 [dependencies]
-openssl-src = { version = "111.22" }
```
Comment (Xuanwo) [inline src/workspace-hack/Cargo.toml:10]: wow!!!!

**Comment by Xuanwo** [comment]
This PR will close https://github.com/datafuselabs/databend/issues/7696

**Comment by Xuanwo** [comment]
This PR is a simple change, let's merge!

**Pull request merged**